typesafe_conductr_cli/test/test_conduct_services.py

```diff
@@ -37,17 +37,17 @@ def test_two_bundles_mult_components_endpoints(self):
 
         http_method.assert_called_with(self.default_url)
         self.assertEqual(
-            strip_margin("""|PROTO  SERVICE           BUNDLE ID  BUNDLE NAME                   STATUS
-                            |http   8010/comp1-endp1  f804d64    multi-comp-multi-endp-1.0.0   Running
-                            |http   8011/comp1-endp2  f804d64    multi-comp-multi-endp-1.0.0   Running
-                            |http   9010/comp2-endp1  f804d64    multi-comp-multi-endp-1.0.0   Running
-                            |http   9010/comp2-endp1  6e4560e    multi2-comp-multi-endp-1.0.0  Running
-                            |http   9011/comp2-endp2  f804d64    multi-comp-multi-endp-1.0.0   Running
-                            |http   6011/comp2-endp2  6e4560e    multi2-comp-multi-endp-1.0.0  Running
-                            |http   7010/comp3-endp1  6e4560e    multi2-comp-multi-endp-1.0.0  Running
-                            |http   7011/comp3-endp2  6e4560e    multi2-comp-multi-endp-1.0.0  Running
+            strip_margin("""|SERVICE                   BUNDLE ID  BUNDLE NAME                   STATUS
+                            |http://:6011/comp2-endp2  6e4560e    multi2-comp-multi-endp-1.0.0  Running
+                            |http://:7010/comp3-endp1  6e4560e    multi2-comp-multi-endp-1.0.0  Running
+                            |http://:7011/comp3-endp2  6e4560e    multi2-comp-multi-endp-1.0.0  Running
+                            |http://:8010/comp1-endp1  f804d64    multi-comp-multi-endp-1.0.0   Running
+                            |http://:8011/comp1-endp2  f804d64    multi-comp-multi-endp-1.0.0   Running
+                            |http://:9010/comp2-endp1  f804d64    multi-comp-multi-endp-1.0.0   Running
+                            |http://:9010/comp2-endp1  6e4560e    multi2-comp-multi-endp-1.0.0  Running
+                            |http://:9011/comp2-endp2  f804d64    multi-comp-multi-endp-1.0.0   Running
                             |
-                            |WARNING: Multiple endpoints found for the following services: /comp2-endp2
+                            |WARNING: Multiple endpoints found for the following services: http://:9010/comp2-endp1
```

Comment:
Roger. I was under the impression that we are going to locate services by the full URI since we moved from `serviceName` to `services` in bundle.conf.